components/pagination/assets/Pagination.scss

```diff
@@ -0,0 +1,55 @@
+@import "@not-govuk/sass-base";
+
+.hods-pagination {
+  &__summary {
+    font-family: 'Inter UI', sans-serif;
+    @include govuk-font(16);
+    -webkit-font-smoothing: antialiased;
+    @include govuk-responsive-padding(2, "bottom");
+    @include govuk-responsive-padding(2, "top");;
+    float: right;
+  }
+
+  &__list-items {
+    display: inline-block;
+  }
+
+  &__item {
+    display: inline-block;
+    list-style: none;
+  }
+
+  &__link {
+    display: inline-block;
+    @include govuk-font(16);
+    @include govuk-responsive-padding(1, "bottom");
+    @include govuk-responsive-padding(1, "top");
+    @include govuk-responsive-padding(2, "left");
+    @include govuk-responsive-padding(2, "right");
+    text-decoration: none;
+
+    &:hover, &:focus {
+      @include govuk-font(16);
+      background: govuk-colour("white");
+      outline: 3px solid govuk-colour("yellow");
+    }
+
+    &-current {
```

Comment:
Do you mean `&--current`?

Reply:
I used `-current` as this is what was in the .tsx file too. Have updated to `--current` in both .tsx file and scss document